llarp/config/config.cpp

```diff
@@ -903,22 +921,26 @@ namespace llarp
             "IP address and use that (with port 1090).",
         });
 
-    conf.defineOption<std::string>(
-        "bind",
-        "*",
-        DefaultOutboundLinkValue,
-        Comment{
-            "Specify a source port for **outgoing** Lokinet traffic, for example if you want to",
-            "set up custom firewall rules based on the originating port. Typically this should",
-            "be left unset to automatically choose random source ports.",
-        },
-        [this](std::string arg) { m_OutboundLink = LinkInfoFromINIValues("*", arg); });
```

Comment:
We need some replacement `defineOption`s here to populate the default generated config with the new versions